pkg/operator/ceph/cluster/mgr/spec.go

```diff
@@ -228,6 +238,22 @@ func (c *Cluster) makeMgrSidecarContainer(mgrConfig *mgrConfig) v1.Container {
 	}
 }
 
+func (c *Cluster) makeCmdProxySidecarContainer(mgrConfig *mgrConfig) v1.Container {
+	_, adminKeyringVolMount := keyring.Volume().Admin(), keyring.VolumeMount().Admin()
+	container := v1.Container{
+		Name:            CommandProxyInitContainerName,
+		Command:         []string{"sleep"},
+		Args:            []string{"infinity"},
```

Comment:
We might want to verify that `sleep` obeys terminal interrupts that allow the pod to restart gracefully

Reply:
I've been thinking about this but didn't want to use `tiny` for this. I've seen my mgr deployment to be updated and pod restarting without issue so I think it handles the interrupts correctly.